### PR TITLE
explicit fail when rocksdb version is too low

### PR DESCRIPTION
**Pull request by biolee**
fix https://github.com/pingcap/tikv/issues/1908

### CODE REVIEW

**Comment by siddontang** [comment]
Thanks @biolee 

PTAL @andelf

File: librocksdb_sys/build.rs [hunk 1]
```diff
@@ -13,7 +13,15 @@ macro_rules! t {
 }
 
 fn main() {
+    let rocksdb_version = "5.4";
```
Comment (andelf) [inline librocksdb_sys/build.rs]: using const will be better

File: librocksdb_sys/build.rs [hunk 2]
```diff
@@ -13,7 +13,15 @@ macro_rules! t {
 }
 
 fn main() {
+    const rocksdb_version: &'static str = "5.4";
```
Comment (andelf) [inline librocksdb_sys/build.rs]: Many thanks.
According to rust style guide, statics and constants use SCREAMING_SNAKE_CASE.
better make it `ROCKSDB_VERSION`

😀

**Comment by siddontang** [comment]
Hi @biolee 
CI failed

**Comment by biolee** [comment]
Edit it on github, did not run on local, sorry for the carelessness

**Comment by biolee** [comment]
I think i should squash the commit

**Pull request closed without merging**